packages/portal/src/modules/contentful/queries/browse-static-page.graphql

```diff
@@ -0,0 +1,183 @@
+query BrowseStaticPage(
+  $identifier: String!
+  $locale: String!
+  $preview: Boolean = false
+) {
+  browsePageCollection(
+    preview: $preview
+    locale: $locale
+    where: { identifier: $identifier }
+    limit: 1
+  ) {
+    items {
+      identifier
+      name
+      headline
+      description
+      image {
+        ...imageFields
+      }
+      hasPartCollection(limit: 20) {
+        items {
+          __typename
+          ... on CardGroup {
+            ...cardGroupFields
+          }
+          ... on AutomatedCardGroup {
+            ...automatedCardGroupFields
+          }
+          ... on ContentTypeRichText {
+            text
+          }
+          ... on PrimaryCallToAction {
+            ...primaryCallToActionFields
+          }
+        }
+      }
+    }
+  }
+  staticPageCollection(
+    preview: $preview
+    locale: $locale
+    where: { identifier: $identifier }
+    limit: 1
+  ) {
+    items {
+      identifier
+      name
+      description
+      image {
+        ...imageFields
+      }
+      hasPartCollection(limit: 40) {
+        total
+        items {
+          __typename
+          ... on ContentTypeRichText {
+            text
+          }
+          ... on Embed {
+            embed
+          }
+          ... on ImageWithAttribution {
+            ...imageWithAttributionFields
+          }
+          ... on ImageComparison {
+            name
+            hasPartCollection(limit: 2) {
+              items {
+                ...imageWithAttributionFields
+              }
+            }
+          }
+          ... on AutomatedCardGroup {
+            ...automatedCardGroupFields
+          }
+          ... on Link {
+            url
+            text
+          }
+          ... on PrimaryCallToAction {
+            ...primaryCallToActionFields
+          }
+        }
+      }
+      relatedLinks {
+        ...linkGroupFields
+      }
+    }
+  }
+
+}
+fragment imageFields on Asset {
+  title
```

Comment:
We no longer need the title field as that was only added for image exceptions which we will now handle using CTF tags.

Reply:
Make sure to remove also from the landing-page query.